shell/mkcd.sh

```diff
@@ -0,0 +1,9 @@
+function_test(){
+    name="$(whoami)"
+    mkdir "$name-folder"
+    cd "$name-folder"
```

Comment:
Maybe we could create directly the file as following : 
`touch "$name-folder/.gitkeep"`